docs/Reference/Rest_API/Rest.md

```diff
@@ -4,11 +4,47 @@ description: Teku RESTful APIa
 
 # REST API
 
-The RESTful JSON HTTP API provides access to beacon chain, node, and network
-information. The API is logically divided into multiple endpoints.
-
-| Endpoint     | Description                                 |
-|--------------|---------------------------------------------|
-| **/beacon**  | General information about the beacon chain. |
-| **/network** | Information about the network.              |
-| **/node**    | Information about the beacon node.          |
+[Enable the REST API service](#enable-the-rest-api-service) from the command line, and enable
+the API documentation endpoint to interact with APIs using Swagger UI.
+
+[Teku REST API documentation] is also available.
+
+## Enable the REST API service
+
+Enable the REST API service by setting the [`--rest-api-enabled`](../CLI/CLI-Syntax.md#rest-api-enabled)
+command line option to `true`.
+
+You can interact with Teku APIs using the web browser by setting the
```

Comment:
I think we could drop _You can_ here to make this more active. ie, 
Interact with Teku APIs ....

Reply:
Done